robottelo/ui/base.py

```diff
@@ -686,6 +686,11 @@ def select(self, target, list_value, wait_for_ajax=True, timeout=30,
         if self.element_type(target) == 'select':
             if isinstance(target, tuple):
                 element = self.wait_until_element(target)
+                if element is None:
```

Comment:
And how does that help? Exception will be raised earlier in `wait_until_element` method and not handled in any way, so you never get to your `if` condition